std_msgs/String windowName
sensor_msgs/Image newImage
bool display
---
